microservice/package/src/main/java/com/cumulocity/agent/packaging/PackageMojo.java

```diff
@@ -105,7 +105,7 @@ public void execute() throws MojoExecutionException {
         if (!containerSkip) {
 
             Iterable<String> buildTargetArchitectures = getTargetBuildArchitectures();
-            log.info("Starting docker microservice build for the following target architectures: {}", List.of(buildTargetArchitectures));
+            log.info("Starting docker microservice build for the following target architectures: {}", Collections.unmodifiableList(new ArrayList<>(Arrays.asList(buildTargetArchitectures))));
```

Comment:
I think we don't need to make this unmodifiable for the logs. it can be simply Arrays.asList

Reply:
I updated the request but I prefer to keep the ArrayList object to keep the print in a nice way.